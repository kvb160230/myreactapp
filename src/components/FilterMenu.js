import React from 'react';

const FilterMenu = ({filterField, filterChange}) => {
    return (
        <div className = 'filter pa1'>
            <select onChange = {filterChange}>
                <option defaultValue="All">All</option>
                <option value="Customer Experience">Customer Experience</option>
                <option value="Engineering">Engineering</option>
                <option value="Operations">Operations</option>
            </select>
        </div>
    
    );
}

export default FilterMenu;