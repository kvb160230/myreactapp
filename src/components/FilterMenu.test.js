import React from 'react';
import { shallow } from 'enzyme';
import FilterMenu from './FilterMenu.js';

it('expect to render Card component', () => {
  const cardComponent = shallow(<FilterMenu />);
  expect(cardComponent.debug()).toMatchSnapshot();
});