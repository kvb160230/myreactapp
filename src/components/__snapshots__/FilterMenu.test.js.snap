// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
"<div className=\\"pa1\\">
  <select onChange={[undefined]}>
    <option defaultValue=\\"All\\">
      All
    </option>
    <option value=\\"Customer Experience\\">
      Customer Experience
    </option>
    <option value=\\"Engineering\\">
      Engineering
    </option>
    <option value=\\"Operations\\">
      Operations
    </option>
  </select>
</div>"
`;
